scenarios/academic/KDD2020-tutorial/README.md

```diff
@@ -0,0 +1,51 @@
+# Environment setup
```

Comment:
the only big issue I have is the location of the files, I think we should move all the content under https://github.com/microsoft/recommenders/tree/master/examples, we can put it as `07_academic` or `07_e2e_tutorials` or whatever name you guys you prefer.

Here the reasoning behind this:
* In scenarios, we only have markdowns, describing business scenarios, where we don't have code. This is targeted to non-technical people interested in recommendation systems
* in examples we have all the notebooks with different modalities of recommendation solutions. This is targeted at technical people that want to learn about reco.
* in reco_utils, we have the libraries and code and support examples. This is targeted to technical people that want to reuse the utilities into their projects

I think this structure is simple and clear, if we start mixing concepts we can end up with a messy repo

Reply:
Done